README.md

```diff
@@ -1,56 +1,33 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
+# Capstone/ Module one
 
+> Capstone projects are solo or group projects at the end of the each of the Microverse Core Curriculum modules. This to allow me to put into practice what have learned during during this periode at Microverse
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML
+- CSS and
+- Javascript
```

Comment:
- [ ] Kindly use github to deploy your web pages and put the live link in readme file. It is a project requirement.